### PR TITLE
KMS and EC2 Automations Outputs

### PR DESCRIPTION
**Pull request by calsupik**
Add tabular and csv file upload outputs for `listKeys` and `listImages`

### CODE REVIEW

File: aws/ec2/listImages/package.json [hunk 1]
```diff
@@ -9,8 +9,10 @@
   "author": "Cal Supik",
   "license": "MIT",
   "dependencies": {
+    "@risk3sixty/extension-sdk": "^0.0.8",
     "aws-sdk": "^2.795.0",
-    "columnify": "^1.5.4"
+    "columnify": "^1.5.4",
+    "moment": "^2.29.1"
```
Comment (whatl3y) [inline aws/ec2/listImages/package.json]: While this is more of a nitpick than anything, I recommend looking into `dayjs` for small projects/utilities for time handling. It's got an identical API as moment but is way more lightweight. It strips out a lot of the locale stuff and what not that moment ships with and its like an order of magnitude or two smaller.

**Review by whatl3y — CHANGES_REQUESTED** [review]
Per our discussion I'd make sure we upload a file, at a minimum a CSV since it will auto populate a request artifact which we want for audit automations.

File: aws/ec2/listImages/package.json (hunk 1 above)
Comment (calsupik) [reply aws/ec2/listImages/package.json]: 7c592c3

**Comment by calsupik** [comment]
> Per our discussion I'd make sure we upload a file, at a minimum a CSV since it will auto populate a request artifact which we want for audit automations.

b0875a9

**Pull request merged**